### PR TITLE
Add rubygem-base64 package

### PR DESCRIPTION
**Pull request by ochnerd**
Add base64 as a dependency for http 5.2.0

### CODE REVIEW

**Comment by dgoetz** [comment]
ok to test

File: packages/plugins/rubygem-base64/rubygem-base64.spec [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+%{gem_libdir}
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+
+%changelog
+* Fri Aug 23 2024  0.2.0-1
```
Comment (dgoetz) [inline packages/plugins/rubygem-base64/rubygem-base64.spec:59]: changelog is missing name and optionally email

File: comps/comps-foreman-el8.xml [hunk 2]
```diff
@@ -405,8 +405,8 @@
       <packagereq type="default">rubygem-google-apis-pubsub_v1-doc</packagereq>
       <packagereq type="default">rubygem-google-apis-sqladmin_v1beta4-doc</packagereq>
       <packagereq type="default">rubygem-google-apis-storage_v1-doc</packagereq>
-      <packagereq type="default">rubygem-googleauth-doc</packagereq>
```
Comment (dgoetz) [inline comps/comps-foreman-el8.xml:408]: We know this from the past, Mac does sorting a bit different then Linux, can you keep the order and only add the new package(s)?

**Review by dgoetz — CHANGES_REQUESTED** [review]
Two changes required see review comments, except from this the built packages look good!

File: packages/plugins/rubygem-base64/rubygem-base64.spec (hunk 1 above)
Comment (ochnerd) [reply packages/plugins/rubygem-base64/rubygem-base64.spec]: Added my name

File: comps/comps-foreman-el8.xml (hunk 2 above)
Comment (ochnerd) [reply comps/comps-foreman-el8.xml]: Reverted sorting and only added my package.

**Review by dgoetz — APPROVED** [review]

**Comment by ekohl** [comment]
In https://github.com/theforeman/foreman-packaging/pull/11097 we had the same discussion and we don't need this for now.

**Comment by ekohl** [comment]
I opened the revert PR in https://github.com/theforeman/foreman-packaging/pull/11173.

**Pull request merged**